pygame_menu/widgets/widget/textinput.py

```diff
@@ -909,6 +917,22 @@ def _check_mouse_collide_input(self, pos):
             topleft, _ = self._rect.topleft
             self._update_cursor_mouse(mousex - topleft)
             return True  # Prevents double click
+        self._cursor_render = True
```

Comment:
Why this? And if helps to fix some issues, why check_touch_collide_input does not force to render again the cursor?

Reply:
It is clearly an error that I may have added when I was trying to see if in needed to copy the _update_cursor_mouse for the touchscreen or use it directly, I'll push a correction.